### ISSUE TITLE
Make it prettier

### ISSUE THREAD

**Issue by thomas-huet** [thomas-huet/aleth#6]
I've done my best but my designing abilities are quite limited.
I welcome all ideas to make the app prettier.

**Comment by lucas-carl** [comment]
Hey, I would like to help you with this issue! I made a first design, and would like to implement it into your app.

![styleguide](https://user-images.githubusercontent.com/13051923/49045595-ca57a300-f1d1-11e8-812a-bb74999fbf3f.png)
![create-edit](https://user-images.githubusercontent.com/13051923/49045592-c9bf0c80-f1d1-11e8-8f5c-7ebf090a394e.png)
![question](https://user-images.githubusercontent.com/13051923/49045594-c9bf0c80-f1d1-11e8-8271-7f157c91e904.png)
![about](https://user-images.githubusercontent.com/13051923/49045591-c9267600-f1d1-11e8-826e-e15a2cf1cb38.png)

Let me know if you like it or not.
Best Regards, Lucas.

**Comment by thomas-huet** [comment]
Hey Lucas,
Thanks for taking the time to do this, it looks great.

Out of curiosity how did you discover my app? I didn't really advertise it (and please don't do it now, I'm actively working on it and may break things in the process).

**Comment by lucas-carl** [comment]
I was searching for issues tagged 'help wanted', just because i was bored :smile:.
I also try to improve my design skills

**Comment by thomas-huet** [comment]
Hi @lucas-carl, do you have some CSS that I could use?

**Comment by lucas-carl** [comment]
@thomas-huet No, i'm sorry. But i could give you the color codes and spacings etc., if you want?

**Comment by thomas-huet** [comment]
Yes please, I can do the rest of the work.
Thanks again.

**Comment by lucas-carl** [comment]
Here you go. I wrote it as SASS variables, so you can use it as your config file.
[config.txt](https://github.com/thomas-huet/aleth/files/2676602/config.txt)

Best regards
Lucas

**Comment by NoneTypez** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
nik13, @PeKaEs, @taglia90, @YarrPiratez, @magick93, @0xRake, @vargaendre, @reznik15, @simasimsd, @shyam-98, @utf, @jrodrigomg, @luijose, @Troush, @scottcheng, @cykor, @sumnerboy12, @IanMadlenya, @mrfelcio, @terrancekk
